src/server/about/controller.js

```diff
@@ -5,6 +5,7 @@
  */
 export const aboutController = {
   handler(_request, h) {
+    _request.logger.info('test')
```

Comment:
so the underscore can be removed here. The underscore previously denoted the arg not being used. Now it is being used we can remove the underscore

Reply:
the "test" log line isn't suppose to be in, keeping underscores and removing log